### PR TITLE
Adjust sync committee size and duration

### PR DESCRIPTION
**Pull request by ralexstokes**
If we merge #2370 then we will also want to tune the sync committee parameters to be friendly to light clients (including bridges, etc.) while still maintaining a floor for security.

### CODE REVIEW

File: configs/mainnet/altair.yaml [hunk 1]
```diff
@@ -22,8 +22,8 @@ INACTIVITY_SCORE_BIAS: 4
 
 # Time parameters
 # ---------------------------------------------------------------
-# 2**8 (= 256)
-EPOCHS_PER_SYNC_COMMITTEE_PERIOD: 256
+# 2**9 (= 512)
+EPOCHS_PER_SYNC_COMMITTEE_PERIOD: 512
```
Comment (djrtwo) [inline configs/mainnet/altair.yaml:26]: We need to force `LIGHT_CLIENT_UPDATE_TIMEOUT` to be equal to the sync committee period.

Tbh, might jsut remove `LIGHT_CLIENT_UPDATE_TIMEOUT` and just use `EPOCHS_PER_SYNC_COMMITTEE_PERIOD` in it's place

**Review by djrtwo — COMMENTED** [review]
Need to adjust or remove the sync protocol value I mentioned below

**Comment by ralexstokes** [comment]
@djrtwo good catch on the `LIGHT_CLIENT_UPDATE_TIMEOUT`, i've addressed that concern and a few other minor things in #2378.

that PR should be merged along w/ this one

File: specs/altair/beacon-chain.md [hunk 2]
```diff
@@ -118,15 +118,15 @@ This patch updates a few configuration values to move penalty parameters toward
...
+| `SYNC_COMMITTEE_SIZE` | `uint64(2**9)` (= 512) |
 | `SYNC_PUBKEYS_PER_AGGREGATE` | `uint64(2**6)` (= 64) |
 | `INACTIVITY_SCORE_BIAS` | `uint64(4)` |
 
 ### Time parameters
 
 | Name | Value | Unit | Duration |
 | - | - | :-: | :-: |
-| `EPOCHS_PER_SYNC_COMMITTEE_PERIOD` | `Epoch(2**8)` (= 256) | epochs | ~27 hours |
+| `EPOCHS_PER_SYNC_COMMITTEE_PERIOD` | `Epoch(2**9)` (= 512) | epochs | ~54 hours |
```
Comment (hwwhww) [inline specs/altair/beacon-chain.md:129]:
```suggestion
| `EPOCHS_PER_SYNC_COMMITTEE_PERIOD` | `uint64(2**9)` (= 512) | epochs | ~54 hours |
```

**Review by hwwhww — APPROVED** [review]

File: configs/mainnet/altair.yaml [hunk 3]
```diff
@@ -12,8 +12,8 @@ PROPORTIONAL_SLASHING_MULTIPLIER_ALTAIR: 2
 
 # Misc
 # ---------------------------------------------------------------
-# 2**10 (= 1,024)
-SYNC_COMMITTEE_SIZE: 1024
+# 2**9 (= 512)
+SYNC_COMMITTEE_SIZE: 512
 # 2**6 (= 64)
```
Comment (Ariya383323) [inline configs/mainnet/altair.yaml:17]: A

File: configs/mainnet/altair.yaml [hunk 4]
```diff
@@ -12,8 +12,8 @@ PROPORTIONAL_SLASHING_MULTIPLIER_ALTAIR: 2
 
 # Misc
 # ---------------------------------------------------------------
-# 2**10 (= 1,024)
-SYNC_COMMITTEE_SIZE: 1024
+# 2**9 (= 512)
+SYNC_COMMITTEE_SIZE: 512
```
Comment (Ariya383323) [inline configs/mainnet/altair.yaml:16]: A

**Review by Ariya383323 — CHANGES_REQUESTED** [review]
I-Level

**Pull request merged**